docs/src/components/landing-page/SponsorLink.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <a
+    :href="`https://${href}`"
```

Comment:
You're duplicating the `https://` here and when providing the link itself. Is the protocol actually needed? You it's usually automatically put there by the browser AFAIK

Reply:
You are right, the browser will add this, and it's preferable leaving it out since their ssl certificate could expire and they go to http